charmcraft/commands/build.py

```diff
@@ -146,12 +146,53 @@ def __init__(self, args, config):
         self.charmdir = args["from"]
         self.entrypoint = args["entrypoint"]
         self.requirement_paths = args["requirement"]
+        self.force_packing = args["force"]
 
         self.buildpath = self.charmdir / BUILD_DIRNAME
         self.ignore_rules = self._load_juju_ignore()
         self.config = config
         self.metadata = parse_metadata_yaml(self.charmdir)
 
+    def show_linting_results(self, linting_results):
+        """Manage the linters results, show some in different conditions, decide if continue."""
+        attribute_results = []
+        lint_results_by_outcome = {}
+        for result in linting_results:
+            if result.result == linters.IGNORED:
+                continue
+            if result.check_type == linters.CheckType.attribute:
+                attribute_results.append(result)
+            else:
+                lint_results_by_outcome.setdefault(result.result, []).append(result)
+
+        # show attribute results
+        for result in attribute_results:
+            logger.debug(
+                "Check result: %s [%s] %s (%s; see more at %s).",
+                result.name,
+                result.check_type,
+                result.result,
+                result.text,
+                result.url,
+            )
+
+        # show warnings (if any), then errors (if any)
+        template = "- %s: %s (%s)"
+        if linters.WARNINGS in lint_results_by_outcome:
+            logger.info("Lint Warnings:")
+            for result in lint_results_by_outcome[linters.WARNINGS]:
+                logger.info(template, result.name, result.text, result.url)
+        if linters.ERRORS in lint_results_by_outcome:
+            logger.info("Lint Errors:")
+            for result in lint_results_by_outcome[linters.ERRORS]:
+                logger.info(template, result.name, result.text, result.url)
+            if self.force_packing:
+                logger.info("Packing anyway as requested.")
+            else:
+                raise CommandError(
+                    "Exiting after lint errors (use --force to pack anyway).", retcode=2
```

Comment:
```suggestion
                    "Exiting after lint errors (use --force to override).", retcode=2
```

Reply:
Maybe `Aborting due to lint errors ...` ?  Exiting is a bit ambiguous.

Reply:
Like it!